marconi-chain-index/test/Spec/Marconi/ChainIndex/Indexers/AddressDatum/AddressDatumIndex.hs

```diff
@@ -341,6 +346,20 @@ propResumingShouldReturnAtLeastOneNonGenesisPointIfStoredOnDisk = property $ do
     resumablePoints <- liftIO $ Storable.resumeFromStorage $ finalIndex ^. Storable.handle
     Hedgehog.assert $ length resumablePoints >= 2
 
+-- | The property verifies that the 'Storable.resumeFromStorage' call returns at least a point which
```

Comment:
I just copy-pasted this from the MintBurn indexer. Ideally, we would have a module for common property tests, but we will work on it later.

Reply:
We have the `marconi-chain-index-test-lib` library component in marconi-chain-index package, maybe it can already be put and imported from there?

Reply:
Yep. Generalized the property and moved it to `test-lib` package.